pkg/operator/profilecalculator.go

```diff
@@ -249,14 +249,14 @@ func (pc *ProfileCalculator) calculateProfileHyperShift(nodeName string) (string
 	for _, recommend := range tunedRecommend(tunedList) {
 		// Start with node/pod label based matching
 		if recommend.Match != nil && pc.profileMatches(recommend.Match, nodeName) {
-			klog.V(2).Infof("calculateProfileHyperShift: node / pod label matching used. node: %s, tunedProfileName: %s, nodePoolName: %s, operand: %v", nodeName, *recommend.Profile, "", recommend.Operand)
+			klog.V(3).Infof("calculateProfileHyperShift: node / pod label matching used for node: %s, tunedProfileName: %s, nodePoolName: %s, operand: %v", nodeName, *recommend.Profile, "", recommend.Operand)
```

Comment:
Appreciate the logging here and below, it is useful and the verbosity is just right.  Thank you.  I should add something like that to `calculateProfile()`.